src/LiteCore/src/LiteCore.Shared/Interop/C4Replicator_defs.cs

```diff
@@ -74,7 +74,25 @@ internal unsafe struct C4Progress
         public ulong documentCount;
     }
 
-	internal unsafe struct C4Replicator
+    unsafe struct C4DocumentReplicatedStatus
+    {
+        public FLSlice docID;
+        public C4Error error;
+        public bool pushing;
+    }
+
+    unsafe struct C4BlobProgress
```

Comment:
Ditto for this one

Reply:
Wow these came in out of order.  It means where is this struct from?

Reply:
BlobProgress in Replicator.hh, but removing it because we don't have it in the eventing feature.